pkg/projects/Microsoft.NETCore.UniversalWindowsPlatform.CoreCLR/Microsoft.NETCore.UniversalWindowsPlatform.CoreCLR.pkgproj

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="dir.props" />
+
+  <PropertyGroup>
+    <Version>2.0.0</Version>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <File Include="files\Microsoft.NETCore.UniversalWindowsPlatform.CoreCLR.props">
```

Comment:
I presume this is the artifact that will also contain UWPShim.exe, right @rahku? If so, @zamont how would that get deployed to the app? Will VS targets take care of it, as they do today?